Refactor code and update documentation (#43)
ballerina-platform/lsp4intellij: README.md

    
    - **RawCommandServerDefinition(string fileExtension, string[] command)** 
        

>         This definition runs the given command. 

Reviewer: ```suggestion
        This definition executes the given command. 
```

Author: Fixed

>         * You can specify multiple extensions for one server by separating them with a comma (e.g., "ts,js").

Reviewer: ```suggestion
        * You can specify multiple extensions for a server by separating them with a comma (e.g., "ts, js").
```

Author: Fixed.

>         * You can specify multiple extensions for a server by separating them with a comma (e.g., "ts,js").

Reviewer: ```suggestion
        * You can specify multiple extensions for a server by separating them with a comma (e.g., "ts, js").
```

    
        * If you want to bind your language server definition only with a specific set of files, you can use that 

>         specific file pattern as a regex expression, instead of binding with the file extension. (e.g., "application*.properties").

Reviewer: ```suggestion
        specific file pattern as a regex expression instead of binding with the file extension (e.g., "application*.properties").
```

Author: Fixed.

>         specific file pattern as a regex expression instead of binding with the file extension. (e.g., "application*.properties").

Reviewer: ```suggestion
        specific file pattern as a regex expression instead of binding with the file extension (e.g., "application*.properties").
```

![](resources/images/lsp4intellij-hover.gif)

#### Workspace Symbols

> Navigate to **Navigate->Symbol...** and enter the symbol name you want to search in the search box popup.

Reviewer: ```suggestion
Navigate to the **Navigate->Symbol...** and enter the name of the symbol you want to search in the search box that pops up.
```
What is **Navigate->Symbol...**? YDid we mean navigate to some place and click "Symbol"?

Author: Please refer the attachment below.

![navigate-symbol](https://user-images.githubusercontent.com/29032600/56257707-c37eed80-60ea-11e9-9263-459e1a7fac96.gif)

![](resources/images/lsp4intellij-hover.gif)

#### Workspace Symbols

> Navigate to the **Navigate->Symbol...** and enter the name of the symbol you want to search in the search box that 

Reviewer: ```suggestion
Click **Navigate** in the top menu, then click **Symbol...**, and enter the name of the symbol you want to search in the search box that 
```

Author: Fixed
